diff --git a/pppd/pppcrypt.c b/pppd/pppcrypt.c
index 193f8fd..b83ebce 100644
--- a/pppd/pppcrypt.c
+++ b/pppd/pppcrypt.c
@@ -106,44 +106,19 @@ Collapse(u_char *in, u_char *out)
 bool
 DesSetkey(u_char *key)
 {
-	u_char des_key[8];
-	u_char crypt_key[66];
-
-	MakeKey(key, des_key);
-	Expand(des_key, crypt_key);
-	errno = 0;
-	setkey((const char *)crypt_key);
-	if (errno != 0)
-		return (0);
-	return (1);
+	return (0);
 }
 
 bool
 DesEncrypt(u_char *clear, u_char *cipher)
 {
-	u_char des_input[66];
-
-	Expand(clear, des_input);
-	errno = 0;
-	encrypt((char *)des_input, 0);
-	if (errno != 0)
-		return (0);
-	Collapse(des_input, cipher);
-	return (1);
+	return (0);
 }
 
 bool
 DesDecrypt(u_char *cipher, u_char *clear)
 {
-	u_char des_input[66];
-
-	Expand(cipher, des_input);
-	errno = 0;
-	encrypt((char *)des_input, 1);
-	if (errno != 0)
-		return (0);
-	Collapse(des_input, clear);
-	return (1);
+	return (0);
 }
 
 #else /* USE_CRYPT */
